index.js

```diff
@@ -48,10 +48,21 @@ function PgSimplifyInflectorPlugin(
     );
   }
 
-  const connectionSuffix = pgOmitListSuffix ? "-connection" : "";
-  const ConnectionSuffix = pgOmitListSuffix ? "Connection" : "";
-  const listSuffix = pgOmitListSuffix ? "" : "-list";
-  const ListSuffix = pgOmitListSuffix ? "" : "List";
+  function omitListSuffix(tag) {
+    return tag != null ? tag === "omit" : !!pgOmitListSuffix;
```

Comment:
Should we thrown if the value is unknown? We'd expect `=== "omit"` or `=== "include"`; all other values should throw I think.

Reply:
I started updating the readme but didn't finish over the weekend. BTW I usually prefer to use arrow functions where I can (my opinion on function hoisting fluctuates) but changed them because it seemed to fit the style used in the file better.

Would you also want to reject `true` as a valid value? That seems reasonable to me but just wanted to make sure.

Since you brought it up I guess throwing is the way to handle this kind of error? I know you have some non-fatal error handling in there, but I don't know how that works.

Reply:
Lets treat `@listSuffix` (i.e. `true`) as if it's `@listSuffix include`. I think that's fairly intuitive. Or throw; I'm happy either way.